kahuna/public/js/components/gr-top-bar/gr-top-bar.css

```diff
@@ -2,7 +2,7 @@ gr-top-bar {
     display: block;
     height: 50px; /* faking we're in the flow */
     position: relative;
-    z-index: 3;
+    z-index: 30;
```

Comment:
Updated a few, I'll make a note to go through and adjust all of them.
For now it works.